packages/cron/src/utils/agrementStatus.ts

```diff
@@ -0,0 +1,14 @@
+// TODO A supprimer et utiliser l'énum partagée une fois la PR de ach mergée
+// Ref: packages/backend/src/dto/AgrementsDto.ts
+export enum AgrementStatut {
```

Comment:
```suggestion
export enum AGREMENT_STATUT {
```
on met les enum en majuscule car ce sont des constantes